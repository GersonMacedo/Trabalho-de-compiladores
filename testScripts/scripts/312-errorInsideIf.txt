! -s CONTENT --disable-logs printer
! should complain about errors inside the if
program p ;
begin
    if false then
        b1 := true
    ;
end .