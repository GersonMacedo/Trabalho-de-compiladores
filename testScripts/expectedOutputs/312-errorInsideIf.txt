
Performing up to content analysis at testScripts/scripts/312-errorInsideIf.txt
ContexAnalyser::var 'b1' at line 6 column 9 was never declared
